### DISCUSSION TITLE
Is this library production ready?

### DISCUSSION THREAD

**Discussion by TowhidKashem** [facebookresearch/playtorch#205] [General]
I ask because the word "demo" and "prototype" is used a lot on the Playtorch website. Is this production ready or still just a toy to test things out with.

If not are there any other reliable on device libraries out there for React Native? Is something like the javascript ONNX runtime more reliable?

Thanks!

**Comment by TowhidKashem** [comment]
Oops when I posted this I didn't notie that it was in beta, will close. Also atm it doesn't work with the latest version of RN, fails during installation so I think it's fair to say it's not stable yet.

**Comment by NafeesMadni** [comment]
Hello how are you
